web/src/components/ProjectsPanel.vue

```diff
@@ -9,109 +9,327 @@
         <input type="checkbox" v-model="selectedProjects" :value="project" />
         <span>{{ project.projectName }}</span>
         <span v-if="project.version">
-          <span class="version-badge">{{
-              project.version
-            }}            
-          </span>             
-          <a  @click="uninstall([project])"><u>Uninstall</u></a>
+          <span class="version-badge">{{ project.version }} </span>
+          <a @click="uninstall([project])"><u>Uninstall</u></a>
         </span>
         <span v-else>
           <span>
-            <a  @click="install([project])"><u>Install</u></a>
+            <a @click="install([project])"><u>Install</u></a>
           </span>
         </span>
       </div>
     </div>
     <loader v-else />
     <div class="package-version-container">
-      <select class="package-version-selector" v-model="selectedVersion">
+      <select
+        class="package-version-selector"
+        v-model="selectedVersion"
+        @change="getInfoPackage()"
+      >
         <option :key="version" v-for="version in versions" :value="version">
           {{ version }}
         </option>
       </select>
-      <button :class="{ disabled: !canInstall() }" @click="install(selectedProjects)">Install</button>
-      <button :class="{ disabled: !canUninstall() }" @click="uninstall(selectedProjects)">Uninstall</button>
+      <button
+        :class="{ disabled: !canInstall() }"
+        @click="install(selectedProjects)"
+      >
+        Install
+      </button>
+      <button
+        :class="{ disabled: !canUninstall() }"
+        @click="uninstall(selectedProjects)"
+      >
+        Uninstall
+      </button>
     </div>
+   
+    <div id="package-description-container">
+      <div class="package-description" v-if="packageMetadata">
+        <h2 class="title-description">Description</h2>
+      
+        <p>{{ packageMetadata.description }}</p>
+
+        <table class="table">
+          <tbody>
+            <tr>
+              <td>Open in:</td>
+              <td>
+                <a target="_blank" :href="packageMetadata.nugetUrl">nuget.org</a>
+              </td>
+            </tr>
+            <tr>
+              <td>Version:</td>
+              <td>{{ packageMetadata.version }}</td>
+            </tr>
+            <tr>
+              <td>Date Published:</td>
+              <td>{{ packageMetadata.datePublished }}</td>
+            </tr>
+            <tr>
+              <td>Author(s):</td>
+              <td>{{ packageMetadata.authors }}</td>
+            </tr>
+            <tr>
+              <td>License:</td>
+              <td>
+                <a v-if="packageMetadata.license !== undefined && packageMetadata.license !== ''" target="_blank" :href="packageMetadata.licenseUrl">{{ packageMetadata.license }}</a>
+                <a v-else target="_blank" :href="packageMetadata.licenseUrl">View license</a>
+              </td>
+            </tr>
+            <tr>
+              <td>Project Url:</td>
```

Comment:
please, add condition to render only if packageMetadata.projectUrl is set, because` packageMetadata.projectUrl.slice(0, 85) `will throw exception in this case